engine/common/src/main/java/org/enso/common/ContextFactory.java

```diff
@@ -187,6 +193,9 @@ public Context build() {
         engineOptions.put(runtimerServerKey, "true");
       }
     }
+    if (pythonHome != null) {
+      options.put("python.PythonHome", pythonHome);
```

Comment:
- setting this property is likely causing premature initialization of `PythonLanguage` 
- when Truffle finds `python.*` option, it must initialize the language
- but in majority of Enso usages, we don't need Python (unless there is `foreign python method` definition)
- how can one tell Python where its home is, but **delay initialization** of the language